### PR TITLE
 Add Japanese translations

### PR DESCRIPTION
**Pull request by Rintan**
I added Japanese translations. I read the translation guideline carefully and obeyed it.

### CODE REVIEW

File: app/src/main/res/values-ja/strings.xml [hunk 1]
```diff
@@ -0,0 +1,389 @@
+<resources>
+    <string name="app_name" translatable="false">FastHub</string>
+    <string name="linear_layout_manager" translatable="false">com.fastaccess.ui.widgets.recyclerview.layout_manager.LinearManager</string>
```
Comment (k0shk0sh) [inline app/src/main/res/values-ja/strings.xml]: Can you please remove all translatable resources from the file

**Comment by k0shk0sh** [comment]
thanks for the translation 👏  .. please remove the mentioned strings and i'll merge it onward.

**Comment by yakov116** [comment]
@Rintan do you have ADA profile? If yes send it over for a mention

**Comment by Rintan** [comment]
@k0shk0sh 
Okay, I'll remove the translatable strings on weekends.
Please wait for new pull request.

**Comment by Rintan** [comment]
@yakov116 
My XDA profile name is Rintan1.

**Comment by yakov116** [comment]
@Rintan 👍

_Sent from my HTC Nexus 9 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by Rintan** [comment]
@yakov116 
Thank you for mentioning on XDA!

**Comment by Rintan** [comment]
@k0shk0sh 
I guess you wanted to say removing the untranslatable strings.
I'll remove them and change some strings then make a new pull request on weekends.

**Comment by JediBurrell** [comment]
@Rintan Basically, If it has `translatable="false"` just remove the line.

**Comment by k0shk0sh** [comment]
@Rintan as @JediBurrell said.
Thanks for contribution :)

**Comment by Rintan** [comment]
@k0shk0sh 
Okay, I'll make the new pull request which include some fixes tomorrow.
Thank you for the everyone's advices!

**Pull request closed without merging**